src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -143,12 +143,15 @@ export class AccountManagementService implements IAccountManagementService {
 		return this.doWithProvider(providerId, async (provider) => {
 			const notificationHandler = this._notificationService.notify(loginNotification);
 			try {
-				let account = await provider.provider.prompt();
-				if (this.isCanceledResult(account)) {
+				let accountResult = await provider.provider.prompt();
+				if (this.isCanceledResult(accountResult)) {
 					return;
+				} else if (this.isErrorResult(accountResult)) {
+					let message = this.getMessage(accountResult);
+					let code = this.getCode(accountResult);
+					throw Error(`\nError Code: ${code}\nError Message: ${message}`);
```

Comment:
Generally putting newlines in an error message isn't the best practice - makes it hard to know how to display an error when it might sometimes contain newlines but sometimes not. I'd just remove them and make the message be displayed on one line.

And is this meant to be shown to the user? Because this needs to be localized if so

Reply:
Yeah the reason I put newlines is to make the formatting nicer for when it is displayed to the user

Reply:
<img width="930" alt="Screenshot 2023-02-22 at 11 51 42 AM" src="https://user-images.githubusercontent.com/18150417/220743272-e26b537a-3d54-4a86-904b-65e8b1707360.png">
this is what it looks like without the newlines, is that okay?

Reply:
The repeated `Error: Error Code` doesn't seem very good, so maybe just `Error: invalid_grant Message: ...`

But if this is immediately being handled by the caller and always displayed then a newline would be fine - so something like 

```
Error: invalid_grant
Message: ....
```

would be easier to read for sure.